### ISSUE TITLE
DNN 9

### ISSUE THREAD

**Issue by vmasanas** [ActiveForums/ActiveForums#471]
Is the latest version released here compatible with DNN 9?

**Comment by WillStrohl** [comment]
Yes.  We're using it on the [DNN community](https://dnncommunity.org) website and you can see the minimum compatible version here:  

https://github.com/DNNCommunity/Dnn.CommunityForums/blob/master/ActiveForums.dnn#L17  

This has been true since version 06.04.00:  

https://github.com/DNNCommunity/Dnn.CommunityForums/releases
